app/packs/src/apps/mydb/elements/list/cellLine/CellLineItemEntry.js

```diff
@@ -48,36 +47,41 @@ export default class CellLineItemEntry extends Component {
   render() {
     const { cellLineItem } = this.props;
     const { currentElement } = ElementStore.getState();
-    let backgroundColor="white-background"
-    if(currentElement) { 
-     backgroundColor=currentElement!==null&&currentElement.id==cellLineItem.id?"blue-background":"white-background";
-    }
+    const backgroundColor = currentElement?.id === cellLineItem.id ? 'blue-background' : 'white-background';
```

Comment:
rename backgroundClass or backgroundColorClass as it represents a css class not a color value